### PR TITLE
Input tooltip will show when a form element is focused.

### PR DESCRIPTION
**Pull request by igoratron**

### CODE REVIEW

**Comment by sgerrand** [comment]
LGTM. :+1:

File: src/sass/general/forms-v2.scss [hunk 1]
```diff
@@ -372,6 +371,7 @@
 
     height: 0;
     padding: 0;
+    display: table-caption;
```
Comment (RazvanDH) [inline src/sass/general/forms-v2.scss]: :+1:

**Comment by RazvanDH** [comment]
LGTM, my only comment is the version: the changes won't break anything, so this should be a small version update (2.4.X)

**Comment by igoratron** [comment]
@RazvanDH done

**Pull request merged**